atd-etl/app/process/queries.py

```diff
@@ -76,11 +76,17 @@ def search_charges(line):
     return line
 
 
-def search_units(line):
+def search_crash_query_full(crash_id, field_list):
     """
-    Generates a graphql query to search for a specific unit.
-    :param line: string - The raw CSV line
+    Generates a graphql query to search for a crash
+    :param crash_id: stirng - The Crash ID to search for.
     :return: string
     """
-    #(p.crash_id, p.unit_nbr)
-    return line
\ No newline at end of file
+    return """
+        query search_crash_query {
+          atd_txdot_crashes(limit: 1, where: {crash_id: {_eq: %CRASH_ID%}}){
+            %FIELD_LIST%
+          }
+        }
+    """.replace("%CRASH_ID%", crash_id)\
+        .replace("%FIELD_LIST%", "\n            ".join(field_list))
```

Comment:
maybe this `.join()` should fall underneath in line with the `.replace`s

Reply:
good catch 👍  after review it should stay where it is, since we are using the output of join to replace the %FIELD_LIST% string.